biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/resolve/BndrunContainer.java

```diff
@@ -323,12 +323,13 @@ public void setEEfromBuild(Run run) {
 						return property;
 					}
 					// so fallback to the currently running Java version
-					return System.getProperty("java.version");
+					return System.getProperty("java.vm.specification.version");
```

Comment:
I think this should be `java.specification.version` rather than `java.vm.specification.version`. See https://docs.oracle.com/javase/8/docs/technotes/guides/versioning/spec/versioning2.html#wp90020.

We use `java.specification.version` elsewhere in Bnd. https://github.com/bndtools/bnd/blob/a92691029d5db01863dfc73f570f1839c9783411/biz.aQute.bndlib/src/aQute/bnd/build/defaults.bnd#L30-L31

Reply:
done!